t/discovery/consul.t

```diff
@@ -576,3 +576,23 @@ GET /thc
 [{"host":"127.0.0.1","port":30513,"priority":0,"weight":1},{"host":"127.0.0.1","port":30514,"priority":0,"weight":1}]
 [{"host":"127.0.0.1","port":30513,"priority":0,"weight":1},{"host":"127.0.0.1","port":30514,"priority":0,"weight":1}]
 --- ignore_error_log
+
+
+
+=== TEST 13: prepare healthy and unhealthy nodes (with consul health check)
```

Comment:
This is just to register or unregister some nodes with consul, so what will happen after using consul service discovery on apisix?

Reply:
I change use consul_dump.t test this

Reply:
So， should delete this one？

Reply:
delete done